### PR TITLE
DSS-PEPPER-362-change-logos-and-pdfs

### PR DESCRIPTION
**Pull request by GiCharkviani**
[PEPPER-362](https://broadworkbench.atlassian.net/jira/software/projects/pepper/boards/164?assignee=621677042d057500723f90e4&selectedIssue=PEPPER-362)

1. Updated Dana Farber logo on the CMI project websites
2. Updated "For your physician" PDFs
3. Updated Brochures added to the MBC, MPC and ESC studies

**Impacted studies:**
- MBC
- MPC
- ASC
- ESC
- OS
- LMS

[PEPPER-362]: https://broadworkbench.atlassian.net/browse/PEPPER-362?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by aweng98** [comment]
I wonder if you could rename `logo-new-DFCI.jpg` to `logo-dana-farber-cancer-institute.png` so that image file names are same acroos all projects. The correct image file is already in the brain project, [logo-dana-farber-cancer-institute.png](https://github.com/broadinstitute/ddp-angular/blob/develop/ddp-workspace/projects/ddp-brain/src/assets/images/logo-dana-farber-cancer-institute.png)
What do you tihink?

File: ddp-workspace/projects/ddp-mbc/src/pdf/For_Your_Physician_MBC_new_DFCI_Logo.pdf
Comment (otchet-broad) [inline ddp-workspace/projects/ddp-mbc/src/pdf/For_Your_Physician_MBC_new_DFCI_Logo.pdf:1]: This file contains multiple languages (Spanish and English).  Is it expected that the english version of the site provide Spanish content first in a PDF?  Might be a question for the ticket reporter.  Is it a trivial lift to split the Spanish and English into different PDFs and download the appropriate one?

**Comment by otchet-broad** [comment]
> I wonder if you could rename `logo-new-DFCI.jpg` to `logo-dana-farber-cancer-institute.png` so that image file names are same acroos all projects. The correct image file is already in the brain project, [logo-dana-farber-cancer-institute.png](https://github.com/broadinstitute/ddp-angular/blob/develop/ddp-workspace/projects/ddp-brain/src/assets/images/logo-dana-farber-cancer-institute.png) What do you tihink?

+1 on this.

File: ddp-workspace/projects/ddp-mbc/src/pdf/For_Your_Physician_MBC_new_DFCI_Logo.pdf
Comment (otchet-broad) [reply ddp-workspace/projects/ddp-mbc/src/pdf/For_Your_Physician_MBC_new_DFCI_Logo.pdf:1]: Whitney says it's okay for now.

**Review by otchet-broad — APPROVED** [review]

**Pull request merged**